examples/cluster-basic.yaml

```diff
@@ -30,6 +30,7 @@ spec:
     httpLoadBalancing: false
```

Comment:
Isn't this on by default ? Looks like it in console, what about API.

Reply:
It's extremely unclear from the API docs but it's on by default in both the UI and API, so this is a bad example here. I'll change it to `true` for this minimal example. The next addon, `networkPolicyConfig`, is disabled in the console and in the API by default, but they obscure that by using the same double-negative `Disabled` setting for both of these.

Reply:
Oh I just realized this was the example file and not the spec. 🤦